cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,55 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+//UpdateConfigHandler returns handler for configuration changes during runtime
+func UpdateConfigHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		_ = req.ParseForm()
+
+		setting := html.UnescapeString(req.Form.Get("setting"))
+		value := html.UnescapeString(req.Form.Get("value"))
+
+		log.Warnf("Got a request to change a setting: %s", setting)
+
+		switch setting {
+		case "log_level":
+			currentLvl, err := log.GetLogLevel()
```

Comment:
We don't need this part, nor do we need to return an error if the requested level is already set.